borgcron_starter.sh

```diff
@@ -57,8 +65,12 @@ case "$1" in
 				"$CURRENT_DIR/borgcron.sh" "$( get_full_path "$CONFIG_DIR/$configfile.sh" )"
 			else
 				echo "The backup settings file \"$configfile.sh\" could not be found." >&2
-				exit 1
+				track_exitcode "$?"
 			fi
 		done
 		;;
 esac
+
+if [ "$exitcode" -gt 0 ]; then
```

Comment:
unnecessary check, if `$exitcode` is 0 then why not do `exit 0`? That's fine. :smiley: